src/rest.js

```diff
@@ -39,6 +39,10 @@ export default class RestClient extends Vue {
     }));
   }
 
+  set token(val) {
+    this.token = val;
```

Comment:
I'm having a hard time wrapping my mind around this, what's going on here? At face value it would seem to be an infinite recursion.

Reply:
Frankly I'm not sure what happened here.  Can we change the private version to `_token`?

Reply:
We could, but what is the purpose of having the setter?

Reply:
https://github.com/girder/demo_site/blob/utm/src/containers/UtmContainer.vue#L39

Reply:
> won't that work without a setter

I just intended to be explicit about allowing external sources to set that property.  In the moment, that seemed like a really important thing to do.  You're right, this can be removed.